src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -285,6 +287,14 @@ void icm426xxGyroInit(gyroDev_t *gyro)
     intfConfig1Value |= ICM426XX_INTF_CONFIG1_AFSR_DISABLE;
     spiWriteReg(dev, ICM426XX_INTF_CONFIG1, intfConfig1Value);
 
+    // If an external 32.768kHz crystal oscillator is connectod, enable it as the timing
+    // source: PIN9_FUNCTION = CLKIN. See Datasheet, section 4.10 for details. Sets
+    // bits 1-2 of the INTF_CONFIG5 register to 0b10.
+    #if defined(CLKIN_EN)
+        uint8_t intfConfig5Value = spiReadRegMsk(dev, ICM426XX_INTF_CONFIG5);
+        intfConfig5Value |= ICM426XX_INTF_CONFIG5_CLKIN;
```

Comment:
```suggestion
        intfConfig5Value  = (intfConfig5Value & ~ICM426XX_INTF_CONFIG5_PIN9_MASK) | ICM426XX_INTF_CONFIG5_CLKIN;
``


Also, I'd rename intfConfig5Value to `ifc5` or similar - it is only 3 lines of code, with register name clearly visible. And you may make it into block, so that the variable does not leak outside at all.